Fix linter warnings

```
$ npm run lint

> @xo-union/union@0.0.1 lint /Users/pkelly/work/union
> eslint --ext .js,.jsx,.json .


/Users/pkelly/work/union/pattern-library/spec/docs/doc-components/PropTypesTable/spec.jsx
  48:5  warning  Duplicate suite: "when attributes are excluded"  jasmine/no-suite-dupes

/Users/pkelly/work/union/pattern-library/spec/docs/doc-components/Snippet/spec.jsx
  26:5  warning  Duplicate spec: "processes the code node with PrismJs"  jasmine/no-spec-dupes

/Users/pkelly/work/union/pattern-library/spec/docs/doc-components/utilities/spec.js
  21:7  warning  Duplicate spec: "returns a readable type format"  jasmine/no-spec-dupes
  32:7  warning  Duplicate spec: "returns a readable type format"  jasmine/no-spec-dupes
  46:7  warning  Duplicate spec: "returns a readable type format"  jasmine/no-spec-dupes

/Users/pkelly/work/union/pattern-library/spec/fields/components/DropdownItem/spec.jsx
  30:7  warning  Duplicate spec: "returns true"  jasmine/no-spec-dupes

/Users/pkelly/work/union/pattern-library/spec/typography/typography-spec.js
  20:5  warning  Duplicate spec: "pulls in the fonts"  jasmine/no-spec-dupes

✖ 7 problems (0 errors, 7 warnings)

```

--- a/.eslintrc.jasmine.js
+++ b/.eslintrc.jasmine.js
@@ -15,6 +15,8 @@ module.exports = {
      * See this:
      * https://gist.github.com/traviskaufman/11131303
      **/
-    'func-names': 'off'
+    'func-names': 'off',
+    'jasmine/no-spec-dupes': [1, 'branch'],
+    'jasmine/no-suite-dupes': [1, 'branch']
   }
 };

--- a/scripts/helpers/diff/logDiffTree.js
+++ b/scripts/helpers/diff/logDiffTree.js
@@ -55,9 +55,11 @@ module.exports = async function logDiffTree() {
   .then(createDiffTreeObject)
   .then(renderTree)
   .then((changedPackages) => {
+    /* eslint-disable */
     console.log('These are changed files since the last publish');
     console.log(`Using tag ${chalk.green(tag)}`);
     console.log(EOL);
     console.log(changedPackages);
+    /* eslint-enable */
   });
 };

--- a/scripts/helpers/diff/parseDiff.js
+++ b/scripts/helpers/diff/parseDiff.js
@@ -69,11 +69,10 @@ module.exports = function parseDiff(diffString) {
   });
 };
 
-module.exports.withColors = function (diffString) {
+module.exports.withColors = function withColors(diffString) {
   return module.exports(diffString).map(({ typeOfChange, name }) => {
     const color = diffStatusColorRules(typeOfChange);
 
     return { name, typeOfChange: new ColoredString(typeOfChange, color) };
   });
 };
-

